### ISSUE TITLE
The error brought by the gt map resize

### ISSUE THREAD

**Issue by yoqim** [davideverona/deep-crowd-counting_crowdnet#5]
HI~
When I calculated the number given by the density map, I found there was a difference between the gt density map before and after downsampling(or resize). So if it is correct to train the model with the downsampled gt density map? Which gt number should be used when evaluating the method? The original number given by the dataset or the number calculated by the downsampled gt density map?

How did you deal with the problem?  Thank you.

**Comment by liangzimei** [comment]
the two number have a little difference, indeed. it is normal. i think the number after resizing is used in the training. 
By the way , can you obtain the mae that @davideverona reported?

**Comment by yoqim** [comment]
Not exactly. After down sampled the density map, I found that there was a big error between the gt number and the summation of the density map. I am trying to figure this out.

**Comment by koalaofpoint** [comment]
excuse me,sir.I want to know how to get the density map by the convnet feature,scaned more blog that they say by a delta function multiple with Gussian konel function,I guess feature map has labeled the local of the head.but the gt density map how to get .and the density map how to get the number of person.I'm very shamed to show so much problem ,but I'm very puzzled @cornicione
